clique/src/manager/mod.rs

```diff
@@ -225,6 +227,14 @@ impl<PK: PublicKey + PeerId, A: Data, D: Data> Manager<PK, A, D> {
             .map_err(|_| SendError::ConnectionClosed)
     }
 
+    pub fn update_metrics<M: NetworkCliqueMetrics>(&self, metrics: &M) {
```

Comment:
Make this a method of `ManagerStatus` instead and call it here. Or perhaps that `ManagerStatus` should be rather returned by that `status_report` method and it should be handled in `service` instead.

Reply:
Done